book/cli/reth/node.md

```diff
@@ -434,6 +434,11 @@ Database:
           - trace:   Enables logging for trace debug-level messages
           - extra:   Enables logging for extra debug-level messages
 
+      --db.exclusive <EXCLUSIVE>
+          Exclusion mode enabled or not
+          
+          [possible values: true, false]
+
```

Comment:
This can be generated with `make update-book-cli`, the description of the argument should match the rustdoc comment.